fontc/src/args.rs

```diff
@@ -31,6 +31,13 @@ pub struct Args {
     #[arg(long, default_value = "false")]
     pub flatten_components: bool,
 
+    /// Whether a component is allowed to have a non-identity 2x2 transform.
+    ///
+    /// If not, any source component that does will be flattened
+    /// Named to match the ufo2ft flag as suggested in <https://github.com/googlefonts/fontc/pull/480#discussion_r1343801553>
```

Comment:
I don't think we need to be reminded of this every time we do `fontc --help`

Reply:
dunno, move it inline with the code somewhere else than the docstring perhaps

Reply:
#485